### DISCUSSION TITLE
PNP search Filter can't find my created column

### DISCUSSION THREAD

**Discussion by sayuri8** [microsoft-search/pnp-modern-search#2753] [Q&A]
Hi everyone,

I have created a list there is a column with section. When I tried to set up the filter possibility, I can't find it in the drop drown. I also tried to find in the result set up but it appears with a strange name. 

This is the column which I have created in the SharePoint list. 
![2B26C5B3-098E-463C-83DB-7BE1A46BF999_4_5005_c](https://user-images.githubusercontent.com/124573411/217254220-e567ab21-8103-4700-8af1-0c2db5d28130.jpeg)

Here is the choice of my filter: 
![AB773F5A-81D1-4F78-B3F2-86DBEF91455F_4_5005_c](https://user-images.githubusercontent.com/124573411/217254578-1af924dc-e41b-46fd-b47e-5db54274965e.jpeg)

This only what it shows:
![078FEA53-6FE4-404B-AF8A-C5E988970045_4_5005_c](https://user-images.githubusercontent.com/124573411/217254093-733de6ac-478c-4dd1-bfd2-c260baea1c3f.jpeg)

**Comment by kasperbolarsen** [comment]
As a generel rule you must map the owl_ [internal fieldname] crawled property to a RefinableString (or RefinableDate is the field is of type Sate) in order to use it as a refiner

**Reply by sayuri8** [reply]
Thanks for your fast response. It means each time when I have created a new column in a list. First that I have to do is to map as you mentioned in your comment? Are there many RefinableString does the type matters like choice, number is it still RefinableString?

**Comment by sayuri8** [comment]
As you mentioned I have mapped but I can't find it in the filter list. One thing that I have to mention in the result drop down I see only from A to C but the other letters I don't see it. 

![87A86CCB-5BB5-4118-B5A6-4A20251812E5](https://user-images.githubusercontent.com/124573411/217269413-88e2dd83-0c96-44c8-b013-d416e0bedcba.jpeg)

**Comment by kasperbolarsen** [comment]
Please read
https://learn.microsoft.com/en-us/sharepoint/manage-search-schema for
understanding the search schema.

And remember you will have to wait up to 24 hours before it takes effect.

**Reply by sayuri8** [reply]
I checked but it doesn't appear in the list. Do I have to unable something else?

**Reply by sayuri8** [reply]
@kasperbolarsen or @wobba I tried to do as you mentioned but I don't see the item which I created as RefinableString.

**Reply by sayuri8** [reply]
I hope someone can help!

**Comment by sayuri8** [comment]
Great thanks!

Oh, I see. Is this only for this the case or also for other settings?
